### DISCUSSION TITLE
After today's update (0.202.14 Hearth & Home), my world will not load

### DISCUSSION THREAD

**Discussion by Joeaculberson** [lloesche/valheim-server-docker#393] [Q&A]
I have a very basic setup on a linux server with pretty much all default settings. This is my first update since I started using this container. 

When I go to login into my world, the game drops me in a completely new world. I've tried coping the backup files into /etc/valheim/worlds, but it just overwrites them when I join the world (except the .old files). It's worth mentioning if I copy these files on to my local machine into C:\Users\<username>\AppData\LocalLow\IronGate\Valheim\worlds, everything loads fine.

If anyone knows how to fix this, I'd really appreciate it. Thanks!

**Comment by findanish** [comment]
I just created a new server but it just wont stop loading after the password screen, doesnt matter if its old or new worlds

**Reply by Joeaculberson** [reply]
Hmm. Sounds like a different situation than mine. Did this just start happening after the update?

**Comment by findanish** [comment]
Yes it started after H&H came out. I bet its because lloesche has not
updated the docker image?

On Fri, Sep 17, 2021 at 4:24 PM Joeaculberson ***@***.***>
wrote:

> Hmm. Sounds like a different situation than mine. Did this just start
> happening after the update?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/lloesche/valheim-server-docker/discussions/393#discussioncomment-1349679>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABYFQNN5RUTKPH7MFT6UEXTUCPE3VANCNFSM5EFJKR5Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Anish Mehta

**Reply by Joeaculberson** [reply]
See my post here: https://github.com/lloesche/valheim-server-docker/issues/394#issuecomment-922123341

It looks like using ghcr.io/lloesche/valheim-server:latest helped me, but that's pure speculation at this point.
